applications/python-the-coffee-bar-apps/src/utils/cpu_increaser.py

```diff
@@ -23,7 +23,7 @@ def outage_start(period: int, threads: int, interval_days: int, start_date: date
         (interval_based_trigger == 'true' and interval_days == 0) or \
         (interval_based_trigger == 'true' and interval_days != 0 and (datetime.strptime(datetime.now().strftime(date_format), date_format) - datetime.strptime(start_date.strftime(date_format), date_format)).days % interval_days == 0):
         log.info('Deploying new version 1.20.124')
-        log.info('Upgrade initiated: admin mode by joe@sumocoffee.com')
+        log.info('Upgrade initiated: admin mode by joe@sumocoffee.com \n')
```

Comment:
Why do we need `\n` here? `log.info` always log in new line

Reply:
we need one more line because sometimes some error occurs in pytz module and in between this info line comes just after that error and  then basically error + this log line in sumo appears as a single message